test/k8s-integration/main.go

```diff
@@ -317,30 +344,37 @@ func handle() error {
 	return nil
 }
 
-func generateTestSkip(normalizedVersion string) string {
+func generateGCETestSkip(clusterVersion string) string {
 	skipString := "\\[Disruptive\\]|\\[Serial\\]"
-	switch normalizedVersion {
-	// Fall-through versioning since all test cases we want to skip in 1.15
-	// should also be skipped in 1.14
-	case "1.13":
-		fallthrough
-	case "1.14":
-		fallthrough
-	case "1.15":
-		fallthrough
-	case "1.16":
-		// "volumeMode should not mount / map unused volumes in a pod" tests a
-		// bug-fix introduced in 1.17
-		// (https://github.com/kubernetes/kubernetes/pull/81163)
+	v := apimachineryversion.MustParseSemantic(clusterVersion)
+
+	// "volumeMode should not mount / map unused volumes in a pod" tests a
```

Comment:
You dropped the "bug-fix introduced in 1.17" line

Reply:
Ack